.dockerignore

```diff
@@ -0,0 +1,5 @@
+.env
+.vscode
```

Comment:
We don't have a `.vscode` folder in this repo.

Reply:
I do have it, it's generated by the IDE

Reply:
It's in `.gitignore` also

Reply:
Yep but it's ignored in the `.gitignore`.

Reply:
Yep, it will never be commited but we have the folder locally in development mode. It's not ver relevant in any case.

Reply:
Yep, probably we didn't create this file in the past because it's not needed for our deployments and this files won't never be included in our deployed instances of the bridge ui. Then, if we are going to have this file I'd include at least all the files in the `.gitignore` to be consistent with our deployment.

Reply:
Done!